core/service/src/util/key_setup/mod.rs

```diff
@@ -1102,15 +1113,18 @@ where
     {
         // Compute signed metadata for CRS verification
         // PANICS: If signature generation fails - would compromise security model
-        let crs_info = compute_info(cur_sk, &DSEP_PUBDATA_CRS, &pp, &domain).unwrap_or_else(|e| {
-            panic!("Failed to compute CRS info for party: {e}");
-        });
+
+        use crate::engine::base::{compute_info_crs, CrsGenMetadata};
```

Comment:
Nitpick: could you import this in the top of the file?

Reply:
moved in dea749b6d314fea2f05a57ce83a4021e337edfc9

Reply:
I cannot see any change to `key_setup/mod.rs` since my last review. Are you sure it is moved?